admob/CMakeLists.txt

```diff
@@ -120,9 +120,13 @@ elseif(IOS)
   # accomplish that.
   symlink_pod_headers(firebase_admob Google-Mobile-Ads-SDK GoogleMobileAds)
 
-  set_target_properties(firebase_admob PROPERTIES
-    FRAMEWORK TRUE
-  )
+  set(TARGET_FORMAT "frameworks" CACHE STRING
```

Comment:
Should this be set in the top level CMake file, and named something like FIREBASE_IOS_TARGET_FORMAT?

Reply:
Sure, let me see if putting it in the top-level CMake file works. Also it's for both iOS and Darwin, so maybe FIREBASE_APPLE_TARGET_FORMAT?

Reply:
or _XCODE_

Reply:
Hmm, XCODE probably makes the most sense, yeah.